### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Create Web Site_Labels or Instructions: Instruction provided for the required fields is not proper as per standards.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#131]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated:** MAS3.3.2

**Repro Steps:** 
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "Create Web Site" expandable button and Press Enter.
6. Tab till "Select Folder" button and Press Enter.
7. Verify that instructions or cues provided for the form fields are provided properly.

**Actual Result:**
For required fields, only red identification mark in the form fields has been provided when it is left blank. there is no "*" mark or "Required" text present with the input fields on the page.

**Expected Result:**
There can be the following standard methods of providing instructions for required field:
1. Using text to indicate required state
2. Using an asterisk to indicate required state
3. Using an image to indicate required state (the image meaning should be defined at the start of the form)

**Note**: This issue is observed throughout the application.

**User Impact:**
Red mark is usually used for identifying the error caused in the form fields. User may not understand that this identification is used to indicate the required fields.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={4b95a70d-258c-474a-820e-5c8b79d5c0bf}

**Attachment:**
[Bug 131_required field identification for the form fields are not proper..pptx](https://github.com/Microsoft/IIS.WebManager/files/2327704/Bug.131_required.field.identification.for.the.form.fields.are.not.proper.pptx)

**Comment by jhkimnew** [comment]
This issue will be fixed when we use WAC SDK controls.

**Comment by shirhatti** [comment]
Don't close these issues. We need them to track the work when we eventually migrate to the WAC controls

**Comment by jhkimnew** [comment]
okay, I will reopen with tagging VNEXT.

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
